### PR TITLE
Set CSP header for index.html

### LINKED ISSUE #43: Implement content security policy headers.
http://www.html5rocks.com/en/tutorials/security/content-security-policy/

### PR DESCRIPTION
**Pull request by pifantastic**
Fixes https://github.com/vole/vole/issues/43

### CODE REVIEW

File: src/vole/vole.go [hunk 1]
```diff
@@ -28,6 +28,18 @@ var userStore = &store.UserStore{
   Version: "v1",
 }
 
+var serveIndex = func(ctx *web.Context) string {
+  ctx.SetHeader("Content-Security-Policy", "script-src 'self' 'unsafe-eval'", true)
```
Comment (marktheunissen) [inline src/vole/vole.go:32]: Oh bummer, yeah we need to yank the handlebars templates out before we can remove the `unsafe-eval` bit. Should aim to do that for sure.

Comment (pifantastic) [inline src/vole/vole.go:32]: Yeah :/

Comment (marktheunissen) [inline src/vole/vole.go:32]: I wonder if we can do this:

`<script type="text/x-handlebars" data-template-name="application" src="/path/to/.hbs>`

With `unsafe-eval` switched on, a lot of the benefits of CSP are lost.

**Comment by marktheunissen** [comment]
Merge this if you like, but then we should open a follow-up issue to remove unsafe-eval.

File: src/vole/vole.go (hunk 1 above)
Comment (pifantastic) [inline src/vole/vole.go:32]: Does that work? Won't Handlebars still eval the response?

Comment (marktheunissen) [inline src/vole/vole.go:32]: I guess we need to precompile them? Will that work?

**Comment by marktheunissen** [comment]
nice one :cake:

File: src/vole/vole.go (hunk 1 above)
Comment (pifantastic) [inline src/vole/vole.go:32]: I know that's possible if we were using requirejs, and if we ran the require optimizer. However, that's only practical in a "production" scenario, don't want that in dev. We could also do some sort of fancy precompilation on the server, but that means porting the Handlebars parser to go :P

**Pull request merged**